app/controllers/games/list.js

```diff
@@ -6,5 +6,10 @@ export default Ember.Controller.extend({
     return !Ember.isNone(this.get('session.user'));
   }.property('session.user'),
 
-  cannotCreateGame: Ember.computed.not('canCreateGame')
+  cannotCreateGame: Ember.computed.not('canCreateGame'),
+
+  /** @property {Array<Game>} all the games that haven't been ended */
+  joinableGames: Ember.computed('model.[].status', function () {
```

Comment:
I'm just not familiar with `array.[].propertyName`. I thought only `array.@each.propertyName` worked. I'm trying the following:

```
var a = [{b:1},{b:2},{b:3}];
a.get('[]'); // [{b:1},{b:2},{b:3}]
a.get('[].b'); // undefined
a.get('@each.b'); // an EachProxy thing
```